Userland/Shell/Builtin.cpp

```diff
@@ -1651,7 +1651,7 @@ ErrorOr<int> Shell::builtin_argsparser_parse(Main::Arguments arguments)
         .value_name = "string",
         .accept_value = [&](StringView value) {
             VERIFY(strlen(value.characters_without_null_termination()) == value.length());
```

Comment:
@alimpfard Is this check still relevant now?

Reply:
No.